template/page-xrp-faucets.html.jinja

```diff
@@ -24,7 +24,8 @@ https://s.devnet.rippletest.net:51234</code></pre>
   <div class="content">
     <p>Ripple runs these <a href="parallel-networks.html">parallel XRP Ledger test networks</a>, the Testnet and Devnet, to provide a testing platform for any software products built on the XRP Ledger without using real funds.</p>
     <p>These funds are intended for <strong>testing</strong> only. The Testnet ledger and balances will be reset approximately every 90 days, or when otherwise necessary. Devnet funds may be reset without warning as needed.</p>
-    <p>Testnet and Devnet balances are separate. As a precaution, do not use the Testnet or Devnet credentials on the Mainnet.</p>
+    <p>Testnet and Devnet balances are separate. As a safety precaution, do not use Mainnet credentials on the Testnet or Devnet and vice-versa.</p>
+    <p>Each faucet request will give out 1000 XRP limited to <strong>testing purposes only</strong></p>
```

Comment:
This is not necessarily true. The amount of test XRP varies by network. Also, as a matter of style, please use present tense ("the faucet gives out") rather than future tense ("the faucet will give out")

Reply:
Should i change the grammar mistake or just delete the whole thing?